src/Components/Common/HeaderPublic.js

```diff
@@ -19,6 +23,11 @@ const HeaderPublic = () => {
          {path: SCHOOL_CAMPAIGN,
          title: "COLEGIOS"},
     ]
+
+    const handleLogout = () => {
+      localStorage.removeItem('token');
+      window.location.reload();
```

Comment:
* Esto esta prohibido en React 😝. Buscar la forma correcta en la cual se podria volver a renderizar el componente.

Reply:
Si creo una constante isLogged con useState  y que la funcion de logout sea :

```
  const handleLogout = () => {
    localStorage.removeItem("token");
    setIsLogged(false);
  };
```
para que se rendericen los botones en base a la constante is Logged, es valido? Teniendo en cuenta que cuando se haga login se deberia cambiar esa constante.